docs/references/expo/use-sso.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: '`useSSO()`'
+description: Clerk's useSSO() hook is used to create a new SSO flow.
+---
+
+The `useSSO()` hook is used to create a new SSO flow. It can be used in both web and native apps.
+
+## Parameters
+
+<Properties>
+  - `strategy`
+
+  - [`OAuthStrategy`](/docs/references/javascript/types/sso#o-auth-strategy)
+
+  - [`EnterpriseSSOStrategy`](/docs/references/javascript/types/sso#enterprise-sso-strategy)
+
+  - `strategy?`
+
+    - `string`
+
+    Select the SSO strategy. The `enterprise_sso` value depends on the object's `identifier` value. Possible `strategy` values are:
+
+    - `oauth_<provider>`: The user will be authenticated with their social sign-in account. [See available social providers](/docs/references/javascript/types/oauth#o-auth-provider).
```

Comment:
This type is only available on `OAuthStrategy`. Did you mean to put that type under that? I think the current formatting makes it look like that `EnterpriseSSOStrategy` has two options: `oauth_<provider>` and `enterprise_sso`.

Reply:
Good catch, let's link to the actual type on each bullet point - updated [here](https://github.com/clerk/clerk-docs/pull/1908/commits/15016025541a0b91dcbe97d4861d21b1593ebc7b)